pkgs/by-name/ha/hactool/package.nix

```
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   installPhase = ''
-    install -D hactool $out/bin/hactool
+    install -D hactool${stdenv.targetPlatform.extensions.executable} $out/bin/hactool${stdenv.targetPlatform.extensions.executable}
```

```suggestion
    install -D hactool${stdenv.hostPlatform.extensions.executable} $out/bin/hactool${stdenv.hostPlatform.extensions.executable}
```

`stdenv.targetPlatform` seems to be overkill here, see https://nix.dev/tutorials/cross-compilation.html#what-s-a-target-platform.

---

Thanks, I guess I misunderstood the difference with host and target.